Source/Extensions/Blazorise.Charts/wwwroot/charts.js

```diff
@@ -111,8 +111,16 @@ function createChart(dotnetAdapter, eventOptions, canvas, canvasId, type, data,
 
     const plugins = [];
 
-    if (pluginNames && pluginNames.includes("dataLabels") && ChartDataLabels) {
-        plugins.push(ChartDataLabels);
+    if (pluginNames) {
+        if (pluginNames.includes("DataLabels") && ChartDataLabels) {
+            plugins.push(ChartDataLabels);
+        }
+
+        if (pluginNames.includes("Streaming")) {
+            plugins.push({
+                duration: 20000
```

Comment:
Should this be configurable?

Reply:
It should. But I will make it part of the new PR.